igcollect/switch.py

```diff
@@ -52,6 +53,25 @@
     'edgeswitch': '.1.3.6.1.4.1.4413.1.1.1.1.4.8.1.3.0',
 }
 
+DOM_OIDS = {
+    'force10_mxl': {
+        'bias_current.0': '1.3.6.1.4.1.6027.3.11.1.3.1.1.18',
+        'bias_current.1': '1.3.6.1.4.1.6027.3.11.1.3.1.1.19',
+        'bias_current.2': '1.3.6.1.4.1.6027.3.11.1.3.1.1.20',
+        'bias_current.3': '1.3.6.1.4.1.6027.3.11.1.3.1.1.21',
+        'rx_power.0': '1.3.6.1.4.1.6027.3.11.1.3.1.1.12',
+        'rx_power.1': '1.3.6.1.4.1.6027.3.11.1.3.1.1.13',
+        'rx_power.2': '1.3.6.1.4.1.6027.3.11.1.3.1.1.14',
+        'rx_power.3': '1.3.6.1.4.1.6027.3.11.1.3.1.1.15',
+        'sfp_voltage': '1.3.6.1.4.1.6027.3.11.1.3.1.1.17',
+        'temperature': '1.3.6.1.4.1.6027.3.11.1.3.1.1.16',
+        'tx_power.0': '1.3.6.1.4.1.6027.3.11.1.3.1.1.8',
+        'tx_power.1': '1.3.6.1.4.1.6027.3.11.1.3.1.1.9',
+        'tx_power.2': '1.3.6.1.4.1.6027.3.11.1.3.1.1.10',
+        'tx_power.3': '1.3.6.1.4.1.6027.3.11.1.3.1.1.11',
```

Comment:
What happens on port with just a single transceiver? I don't think we have any 100Gb/s ports on Dell switches.

Reply:
We are not talking about transceivers, but about lanes on the transceivers. 

Ports with transceivers with a single lane will output an empty value for the other lanes that gets ignored on line 371:
```python
                if data and data != '':
                    print(template.format(port_name, metric, data))
```